plugins/wallet_plugin/wallet_manager.cpp

```diff
@@ -0,0 +1,164 @@
+#include <eos/wallet_plugin/wallet_manager.hpp>
+#include <eos/utilities/key_conversion.hpp>
+
+namespace eos {
+namespace wallet {
+
+constexpr auto file_ext = ".wallet";
+constexpr auto password_prefix = "PW";
+
+std::string gen_password() {
+   auto key = fc::ecc::private_key::generate();
+   return password_prefix + utilities::key_to_wif(key);
+
```

Comment:
Note here I'm creating a password for the user by creating a private key and prefixing it with "PW" so a user can easily distinguish it from other keys.